go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -877,3 +901,34 @@ func (bsnc *BaseSecondaryNetworkController) requireDHCP(pod *corev1.Pod) bool {
 		bsnc.IsPrimaryNetwork() &&
 		bsnc.TopologyType() == types.Layer2Topology
 }
+
+func (bsnc *BaseSecondaryNetworkController) setPodLogicalSwitchPortEnabledField(
```

Comment:
yes, in case of failed migration (see last commit `podNetworkController: Switch back source pod LSP after failed migration`) - we need to transact the changes of the Source-pod LSP.